templates/phaser/packages/client/src/mud/createSystemCalls.ts

```diff
@@ -6,10 +17,33 @@ import { singletonEntity } from "@latticexyz/store-sync/recs";
 export type SystemCalls = ReturnType<typeof createSystemCalls>;
 
 export function createSystemCalls(
+  /*
+   * The parameter list informs TypeScript that:
+   *
+   * - The first parameter is expected to be a
+   *   SetupNetworkResult, as defined in setupNetwork.ts
+   *
+   * - Out of this parameter, we only care about two fields:
+   *   - worldContract (which comes from createContract, see
+   *     https://github.com/latticexyz/mud/blob/main/templates/phaser/packages/client/src/mud/setupNetwork.ts#L31).
+   *   - waitForTransaction (which comes from syncToRecs, see
+   *     https://github.com/latticexyz/mud/blob/main/templates/phaser/packages/client/src/mud/setupNetwork.ts#L39).
```

Comment:
I prefer to have them because it lets people look for additional information if they are interested. Also, it helps transition them into looking at the source code.